pydantic_ai_slim/pydantic_ai/durable_exec/temporal/_agent.py

```diff
@@ -83,6 +85,14 @@ def __init__(
         Args:
             wrapped: The agent to wrap.
             name: Optional unique agent name to use in the Temporal activities' names. If not provided, the agent's `name` will be used.
+            models:
+                Optional mapping of models to register with the agent.
+                Keys define the names that can be referenced at runtime and the values are `Model` instances.
+                Registered model instances can be passed directly to `run(model=...)` and will be resolved to their key.
```

Comment:
"will be resolved to their key" is not relevant to the user is it? I think we should just say that both the key and the instance can be used on `run(model=...)`